xblock/mixins.py

```diff
@@ -538,3 +538,50 @@ def index_dictionary(self):
         default implementation is an empty dict
         """
         return {}
+
+
+class ViewsMixin(object):
+    """
+    This mixin provides decorators that can be used on xBlock view methods.
+    """
+    @classmethod
+    def supports(cls, functionality):
```

Comment:
Yeah, it would be nice to keep them consistent. If you feel like making the change as part of this PR, that would be awesome, otherwise we can just leave them all as single-argument for now.